site/content/docs/5.3/examples/download-app/index.html

```diff
@@ -0,0 +1,373 @@
+                <use xlink:href="#tick"/>
+              </svg>
+              <p class="fw-bold mb-0">In situ visibility of all the elements available in the mobile code library for each mobile operating system</p>
+            </li>
+            <li class="d-flex align-items-start mb-3">
+              <svg xmlns="http://www.w3.org/2000/svg" class="me-3 flex-shrink-0" width="1.875rem" height="1.875rem" role="img" focusable="false" aria-hidden="true">
+                <use xlink:href="#tick"/>
+              </svg>
+              <p class="fw-bold mb-0">Live experience of the available interactions for each component, and ability to select the right customisation</p>
+            </li>
+            <li class="d-flex align-items-start mb-3">
+              <svg xmlns="http://www.w3.org/2000/svg" class="me-3 flex-shrink-0" width="1.875rem" height="1.875rem" role="img" focusable="false" aria-hidden="true">
+                <use xlink:href="#tick"/>
+              </svg>
+              <p class="fw-bold mb-0">Time saving by finding the code to use, by selecting the right component variant and it's possible customisation</p>
+            </li>
+            <li class="d-flex align-items-start mb-3">
+              <svg xmlns="http://www.w3.org/2000/svg" class="me-3 flex-shrink-0" width="1.875rem" height="1.875rem" role="img" focusable="false" aria-hidden="true">
+                <use xlink:href="#tick"/>
+              </svg>
+              <p class="fw-bold mb-0">Confidence in the fact that the available components are following the Orange brand standard</p>
+            </li>
+            <li class="d-flex align-items-start mb-3">
+              <svg xmlns="http://www.w3.org/2000/svg" class="me-3 flex-shrink-0" width="1.875rem" height="1.875rem" role="img" focusable="false" aria-hidden="true">
+                <use xlink:href="#tick"/>
+              </svg>
+              <p class="fw-bold mb-0">Assurance that the components have been tested amongst the accessibility standard from the industry</p>
+            </li>
+            <li class="d-flex align-items-start mb-md-3">
+              <svg xmlns="http://www.w3.org/2000/svg" class="me-3 flex-shrink-0" width="1.875rem" height="1.875rem" role="img" focusable="false" aria-hidden="true">
+                <use xlink:href="#tick"/>
+              </svg>
+              <p class="fw-bold mb-0">Coverage of iOS and Android with app and SDK being available in open-source</p>
+            </li>
+          </ul>
+        </div>
+        <div class="col-12 col-md-6 offset-md-1 mt-2 mt-md-0">
+          <div class="mastermedia-player d-flex justify-content-center align-items-center w-100 h-100 border-dark border border-1" videoid="pmJLYXq6kC" title="Explanation on how to get the app on your phone" classlist="w-100 h-100 border-dark border border-1">
```

Comment:
We shouldn't explain in the title that it's a video?

Reply:
Tried something in the last commit.